bcda/models/models_test.go

```diff
@@ -836,3 +836,37 @@ func (s *ModelsTestSuite) TestCMSID() {
 	assert.Equal(s.T(), 1, len(actualCMSID))
 	assert.Equal(s.T(), cmsID, actualCMSID[0])
 }
+
+func (s *ModelsTestSuite) TestCCLFFileType() {
+	noType := &CCLFFile{
```

Comment:
Should we add one more test to include explicitly defining the type `FileTypeDefault`?

Reply:
> Should we add one more test to include explicitly defining the type `FileTypeDefault`?

The `noType` here should verify the `FileTypeDefault` case (since the zero value for CCLFFile is `FileTypeDefault`).